core-v1/src/main/scala/lens/VariantLens.scala

```diff
@@ -246,8 +254,16 @@ final case class VariantLens[-A, +B](
     * @note this would not be necessary if [[VariantLens]] used appropriate variance, however, variance for lenses
     *       introduces a lot of issues.
     */
-  def as[V](implicit ev: B <:< V): VariantLens[A, V] =
-    this.copy(get = this.get.andThen(ev))
+  def as[V](implicit convert: B => V): VariantLens[A, V] =
+    this.copy(get = this.get.andThen(convert))
```

Comment:
An implicit function is also know as a "view bound". It had its own syntax `def f[A <% B](a: A)` that was deprecated in Scala 2.11 in favor of just accepting the implicit function argument `def f[A](a: A)(implicit ev: A => B)`.